### PR TITLE
migration to add timestamps to crawl receipts

### PR DESCRIPTION
**Pull request by Jeremy-Xue**
in order to support ordering by created_at timestamp

### CODE REVIEW

File: db/migrate/20210129033100_add_timestamps_to_crawl_receipts.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class AddTimestampsToCrawlReceipts < ActiveRecord::Migration[6.0]
+  def change
+    change_table(:crawl_receipts) { |t| t.timestamps}
```
Comment (jacoblb64) [inline db/migrate/20210129033100_add_timestamps_to_crawl_receipts.rb:3]: huh, didn't know `timestamps` was defined like this, cool!

**Review by jacoblb64 — APPROVED** [review]
Looks great to me! I think you have some unit test failures to fix though

File: db/migrate/20210129033100_add_timestamps_to_crawl_receipts.rb (hunk 1 above)
Comment (Jeremy-Xue) [reply db/migrate/20210129033100_add_timestamps_to_crawl_receipts.rb:3]: Had to wait for the timecop gem pr to land before being able to fix the unit tests! Fixing them now 👍🏼

**Comment by Jeremy-Xue** [comment]
not sure why tests are failing, going to look at this again tomorrow

**Pull request merged**